adapters/ddf/src/main/java/com/connexta/replication/adapters/ddf/DdfNodeAdapterFactory.java

```diff
@@ -77,8 +84,12 @@
 
   private ClientFactoryFactory clientFactoryFactory;
 
-  public ReplicationStore createReplicatorStore(URL url) {
+  public DdfNodeAdapterFactory(DdfRestClientFactory ddfRestClientFactory) {
+    this.ddfRestClientFactory = ddfRestClientFactory;
+  }
 
+  @Override
+  public NodeAdapter create(URL url) {
     CswSourceConfiguration cswConfiguration = new CswSourceConfiguration(encryptionService);
     cswConfiguration.setCswUrl(url.toString() + "/services/csw");
```

Comment:
❓ Are we guaranteed the root context path is always `/services`? I thought it was customizable?

Reply:
No it's not a guarantee, it's able to be customized. We make these assumptions for the REST and CSW endpoints.

Reply:
🤔

Reply:
Handled here: https://github.com/connexta/replication/pull/81